libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/FlyoutObjectDetection.tsx

```diff
@@ -201,11 +184,11 @@ export class FlyoutObjectDetection extends React.Component<
                     </Stack>
                   </Stack.Item>
                   <Stack.Item className={classNames.imageContainer}>
-                    <Image
-                      src={`data:image/jpg;base64,${item?.image}`}
-                      className={classNames.image}
-                      imageFit={ImageFit.contain}
-                    />
+                    <Stack.Item id="canvasToolsDiv">
```

Comment:
nit -do we want to abstract these strings to the utils file, and replace with constants?